apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -8,51 +8,160 @@ import CourseCatalogBannerBackground from '../../../static/curriculum_catalog/co
+              checkedOptions={appliedFilters[filterType.name]}
+              onChange={e => handleSelect(e, filterType.name)}
+            />
+          );
+        })}
+        <button
+          id="clear-filters"
+          type="button"
+          className={style.catalogClearFiltersButton}
+          onClick={handleClear}
+        >
+          Clear
+        </button>
       </div>
-    </div>
-  </>
-);
+      <div className={style.catalogContentContainer}>
+        <div className={style.catalogContent}>
+          {/*TODO [MEG]: calculate and pass in duration and translated from backend */}
+          {curriculaData
+            .filter(curriculum => !!curriculum.grade_levels)
+            .map(
+              ({
+                key,
+                image,
+                display_name,
+                grade_levels,
+                school_subject,
+                cs_topic,
+              }) => (
+                <CurriculumCatalogCard
+                  key={key}
+                  courseDisplayName={display_name}
+                  imageSrc={image || undefined}
+                  duration={'school_year'} // TODO [MEG] actually pass in this data
+                  gradesArray={grade_levels.split(',')}
+                  subjects={school_subject?.split(',')}
+                  topics={cs_topic?.split(',')}
+                  isTranslated={false} // TODO [MEG]: actually pass in this data
+                  isEnglish={isEnglish}
+                />
+              )
+            )}
+        </div>
+      </div>
+    </>
+  );
+};
 
 CurriculumCatalog.propTypes = {
   curriculaData: PropTypes.arrayOf(curriculumDataShape),
   isEnglish: PropTypes.bool.isRequired,
 };
 
+const FilterCheckboxDropdown = ({
+  name,
+  label,
+  allOptions,
+  checkedOptions,
+  onChange,
+}) => {
+  return (
+    <div id={`${name}-dropdown`} className="dropdown">
+      <button
+        id={`${name}-dropdown-button`}
+        type="button"
+        className="selectbox"
+        data-toggle="dropdown"
+      >
+        {label}
+      </button>
+      <ul className="dropdown-menu">
+        <form>
+          {allOptions.map(option => {
+            return (
+              <li key={`${name}-${option}`} className="checkbox form-group">
+                <input
+                  type="checkbox"
+                  id={`${name}-${option}-check`}
+                  name={option}
+                  value={option}
+                  checked={checkedOptions.includes(option)}
+                  onChange={onChange}
+                />
+                <label htmlFor={`${name}-${option}-check`}>{option}</label>
+              </li>
+            );
+          })}
+        </form>
+      </ul>
+    </div>
+  );
+};
+FilterCheckboxDropdown.propTypes = {
+  name: PropTypes.string,
+  label: PropTypes.string,
+  allOptions: PropTypes.arrayOf(PropTypes.string),
+  checkedOptions: PropTypes.arrayOf(PropTypes.string),
+  onChange: PropTypes.func,
+};
```

Comment:
the `FilterCheckboxDropdown` component looks like a good candidate to extract into its own file.